foundation_cms/legacy_apps/donate_banner/models.py

```diff
@@ -178,7 +181,7 @@ def clean(self):
             )
 
 
-class DonateBannerPage(Page):
+class DonateBannerPage(FoundationNavigationPageMixin, Page):
```

Comment:
Updated to utilize `FoundationNavigationPageMixin`'s new "check whether I live under the legacy or redesign" homepage functionality.